openfisca_france_dotations_locales/parameters/dotation_solidarite_rurale/perequation/attribution/plancher_ratio_progression.yaml

```
@@ -0,0 +1,5 @@
+description: Plancher de l'attribution de la DSR fraction péréquation pour une commune éligible en fonction de son attribution de l'année précédente
+reference: https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000036433094&cidTexte=LEGITEXT000006070633
+values:
+  2019-01-01:
```

De même que pour le plafond péréquation : 
```suggestion
  2012-01-01:
```